dbutils.py

```diff
@@ -0,0 +1,18 @@
+import os
+import psycopg2
+
+db_user = os.environ["DB_USER"]
+db_pass = os.environ["DB_PASS"]
+db_name = os.environ["DB_NAME"]
+db_host = os.environ["DB_HOST"]
+cloud_sql_connection_name = os.environ["CLOUD_SQL_CONNECTION_NAME"]
+
```

Comment:
Nice. Share me the .env file through mail where your environment variables are located so that I can reproduce it in my local environment.